DSCResources/cChocoInstaller/cChocoInstaller.psm1

```diff
@@ -229,7 +232,7 @@ function InstallChoco
     
     # unzip the package
     Write-verbose "Extracting $file to $tempDir..."
-    Start-Process "$7zaExe" -ArgumentList "x -o`"$tempDir`" -y `"$file`"" -Wait
+    Start-Process "$7zaExe" -ArgumentList "x -o`"$tempDir`" -y `"$file`"" -Wait -NoNewWindow
```

Comment:
Pulled down the merge and run some tests, all seems to be well. 

The original reason for including the installer script in the DSC was to refactor the use of Write-Host as it isn't support when running under DSC. 

I'll look to revisit as may be a more elegant way to work around this to have it pull down the latest installer from chocolatey.org, which would be much nicer.